sw/device/examples/hello_usbdev/hello_usbdev.c

```diff
@@ -22,7 +22,7 @@
 #include "hw/top_earlgrey/sw/autogen/top_earlgrey.h"  // Generated.
 
 // These just for the '/' printout
-#define USBDEV_BASE_ADDR 0x40150000
+#define USBDEV_BASE_ADDR TOP_EARLGREY_USBDEV_BASE_ADDR
```

Comment:
Nit: you could just use the `TOP_EARLGREY_USBDEV_BASE_ADDR` at call sites.

Reply:
Oh yeah, this definition was originally for the old version of the generated headers, which we no longer have to support. This is probably better as a separate cleanup PR rather than a quick fix PR.